src/main/java/Config.java

```diff
@@ -33,6 +34,7 @@ public static void init() throws Exception {
 
         READINESS_TOPIC = getOptionalString(dotenv, "READINESS_TOPIC", null);
         LINGER_TIME_MS = getOptionalInt(dotenv, "LINGER_TIME_MS", 0);
+        LINGER_TIME_MS = getOptionalString(dotenv, "BATCH_SIZE", null);
```

Comment:
BATCH_SIZE

Reply:
Ooops, fixed - can you approve the tests workflow?